CODEOWNERS

```diff
@@ -0,0 +1,40 @@
+# This is a comment.
```

Comment:
@qvistgaard I think you overwrite my commit with the changes I made to this file.

Anyway, could you move lines 8-10 to `.github/CODEOWNERS` and delete this file altogether?